examples/time_series_anomaly_demo.py

```
@@ -0,0 +1,51 @@
+"""Simple demo for time-series anomaly detection using SignalProcessor."""
+
+import numpy as np
+import matplotlib.pyplot as plt
```

Ruff (0.11.9)

`matplotlib.pyplot` imported but unused

Remove unused import: `matplotlib.pyplot`

(F401)